renderer/components/editor/controls/right.js

```diff
@@ -7,18 +8,27 @@ import {connect, VideoContainer, EditorContainer} from '../../../containers';
 import formatTime from '../../../utils/format-time';
 
 class RightControls extends React.Component {
+  state = {}
+
+  componentDidMount() {
+    const settings = electron.remote.require('./common/settings');
```

Comment:
@karaggeorge Do we have the information about if this recording was recorded with audio or not here? Since loading the setting when the controls mount, you could have changed the settings between doing the recording and this happening, for example when opening up an existing file or an old recording via the export UI.

Reply:
I have no idea, first time contributing to this repo :)

The editor is mounted once the recording is done so i'm not sure you can change the settings between the recording and the moment the editor is mounted

Reply:
Then we need the video player to give us that settings right ? But how do i pass the video settings from tag to the control component, from what i can see the video component and controls are at the same depth in video player and Kap has no global store from what i could see.

I'm not sure i can have access to the video file in the controls to check is audio is available or not. If this is possible, let me know and i will update the PR :)

Reply:
I will have a look tonight how it could be done and give you some pointers :)

Reply:
I would look into something like this: 
https://stackoverflow.com/questions/21446804/find-if-video-file-has-audio-present-in-it

I used ffmpeg to find the fps of a video (which is set to the max exported fps). If we could do something like that at the same spot (I think there's a util file and then it's used in the editor file in the main process) that would probably cover all the cases

Reply:
I saw you might be interested in helping out with this issue @hooskers, feel free to chime in!

Reply:
According to https://stackoverflow.com/a/31022391/64949, we can use this to detect whether the video has audio:

```js
const hasAudio = video => {
	return video.webkitAudioDecodedByteCount > 0 ||
	Boolean(video.audioTracks && video.audioTracks.length > 0);
};
```